R/nested.R

```diff
@@ -37,6 +36,18 @@ compute_parse_data_nested <- function(text) {
   pd_nested
 }
 
+#' Obtain token from text
+#'
+#' [utils::getParseData()] is used to obtain a flat parse table from `text`.
+#' @param text A character vector.
+#' @return A flat parse table
+tokenize <- function(text) {
```

Comment:
Do we have another instance of `getParseData()` that can be replaced with a `tokenize()` call?